pipeline/lib/pipeline.ml

```diff
@@ -9,6 +9,9 @@ module Benchmark = Models.Benchmark
 
 let ( >>| ) x f = Current.map f x
 
+let is_duplicate elem lst = lst |>
```

Comment:
Instead of checking this in the list every time, can we just have a hashtable and check if the value exists there? It'll be faster and there won't be a lot of names so not a memory problem either.